racket/src/worksp/README.txt

```diff
@@ -21,6 +21,10 @@ tries to find Visual Studio and run its "vcvarsall.bat". Provide an
 argument such as "x86" (32-bit mode) or "x86_amd64" (64-bit mode) to
 select the build mode.
 
+When using Power Shell, use the "msvcprep.ps1" script instead of
+"msvcprep.bat".  Running the latter from PS will appear to work, but
+will not actually change any environment variables.
```

Comment:
Nit: "Power Shell" should be "PowerShell". Also, I'd use "PowerShell" instead of the "PS" abbreviation just once.